primitives/externalities/src/lib.rs

```diff
@@ -156,6 +156,16 @@ pub trait Externalities: ExtensionStore {
 	///
 	/// Returns the SCALE encoded hash.
 	fn storage_changes_root(&mut self, parent: &[u8]) -> Result<Option<Vec<u8>>, ()>;
+
+	/// Create a new transactional layer.
+	fn storage_start_transaction(&mut self);
+
+	/// Discard a transactional layer, pending changes of every transaction behind this layer are
+	/// dropped (including committed changes) .
```

Comment:
```suggestion
	/// dropped (including committed changes).
```

please pay attention to detail.